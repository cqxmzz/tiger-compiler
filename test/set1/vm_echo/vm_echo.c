#include "vmgrader.h"
//
// Cheng Yu
//
char* argv[3];

int main() {
   int status;
   argv[0] = "nachos";
   argv[1] = "testing";
   argv[2] =  "echo.";
   join(exec("echo.coff", 3, argv), &status);
   if(status != 0) {
       printf("join failed.\n");
       fail();
   }
   done();
   return 0;
}
