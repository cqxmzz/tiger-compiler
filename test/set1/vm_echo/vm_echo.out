3 arguments
arg 0: nachos
arg 1: testing
arg 2: echo.
